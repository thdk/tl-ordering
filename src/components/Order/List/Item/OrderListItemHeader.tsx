import * as React from "react";

export const OrderListItemHeader: React.FunctionComponent = () => {
    return (
        <div className="row header">
            <div className="row-left">
                <div>Order id</div>
                <div>Customer id</div>
            </div>
            <div className="row-right">
                <div>Total</div>
                <div>
                    Order detail
                </div>
            </div>
        </div>
    );
};

OrderListItemHeader.displayName = "OrderListItemHeader";

export default OrderListItemHeader;
