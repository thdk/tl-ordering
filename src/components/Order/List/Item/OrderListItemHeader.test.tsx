import { shallow, ShallowWrapper } from "enzyme";
import React from "react";
import OrderListItemHeader from "./OrderListItemHeader";

describe("Render OrderListItemHeader", () => {
    let component: ShallowWrapper;

    beforeEach(() => {
        component = shallow(
            <OrderListItemHeader />,
        );
    });

    it("should match snapshot", () => {
        expect(component.debug()).toMatchSnapshot();
    });
});
