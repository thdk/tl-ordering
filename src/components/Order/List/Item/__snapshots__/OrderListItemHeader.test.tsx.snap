// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render OrderListItemHeader should match snapshot 1`] = `
"<div className=\\"row header\\">
  <div className=\\"row-left\\">
    <div>
      Order id
    </div>
    <div>
      Customer id
    </div>
  </div>
  <div className=\\"row-right\\">
    <div>
      Total
    </div>
    <div>
      Order detail
    </div>
  </div>
</div>"
`;
